runtime/acala/src/lib.rs

```diff
@@ -178,7 +179,10 @@ pub fn get_all_module_accounts() -> Vec<AccountId> {
 		TreasuryPalletId::get().into_account(),
 		TreasuryReservePalletId::get().into_account(),
 		UnreleasedNativeVaultAccountId::get(),
-	]
+		TreasuryFeePoolPalletId::get().into_account(),
+	];
+	whitelist.append(&mut get_all_sub_account_of_fee_pool());
+	whitelist
```

Comment:
rollback to disable treasury aub account. make sure the sub account has ED of foreign asset in initial_pool phase.